rules/mv.go

```diff
@@ -0,0 +1,48 @@
+package rules
+
+import (
+	"github.com/jessevdk/go-flags"
+	"github.com/nopecmd/nope/match"
+	"github.com/nopecmd/nope/models"
+	"github.com/nopecmd/nope/shells"
+	"strings"
+)
+
+var mvFlags struct {
+	Interactive bool `short:"i"`
+	Force       bool `short:"f"`
+}
+
+const (
+	mvBaseCommand = "mv"
+)
+
+func isMatchMv(cmd models.Command) bool {
+	return cmd.BaseCommand == mvBaseCommand
+}
+
+func getUndoMv(cmd models.Command) string {
+	// for now, just remove all flags
+	var err error
+	cmd.TokensWithoutFlags, err = flags.ParseArgs(&mvFlags, cmd.Tokens[1:])
+
+	if err != nil {
+		panic(err)
+	}
+
```

Comment:
That's what I was originally saying with moving this into parsing. It doesn't really belong in match, because it's not doing matching... We need to revisit this.